Formula/gauge.rb

```diff
@@ -43,6 +43,6 @@ def install
   end
 
   test do
-    assert_match version.to_s, shell_output("#{bin}/gauge -v")
+    assert_match version.to_s[0, 5], shell_output("#{bin}/gauge -v")
```

Comment:
`gauge -v` shows version in the format `MAJOR.MINOR.PATCH` which is `0.8.0` but the version in this tag is `0.8.0.1`. So, I am using this to match `0.8.0.1` with `0.8.0`(`MAJOR.MINOR.PATCH`).